source/documentation/guidance/upgrading-from-cflinuxfs2.md

```diff
@@ -0,0 +1,25 @@
+## Upgrading from cflinuxfs2
+
+When you run an app on the PaaS using a buildpack, it runs on top of a version of Ubuntu Linux. We are upgrading the version of Ubuntu used from Ubuntu Trusty to Ubuntu Bionic. In Cloud Foundry, this takes the form of upgrading the stack your apps use from `cflinuxfs2` to `cflinuxfs3`.
+
+Ubuntu Trusty is no longer getting security updates and to ensure the security of the platform we will need to forcibly upgrade all apps. It is possible that this will break your application. You can prevent this happening by manually upgrading.
+
+To check if you are affected by this change, log in to your admin panel for [Ireland](https://admin.cloud.service.gov.uk) or [London](https://admin.london.cloud.service.gov.uk). Click on each organisation you are a part of. If you are affected then the interface will warn you and list the affected apps.
```

Comment:
Minor tweaks for style and consistency of language:

"To check if you are affected by this change, sign in to your GOV.UK PaaS admin tool for [Ireland](https://admin.cloud.service.gov.uk) or [London](https://admin.london.cloud.service.gov.uk). Select each of your organisations. If you are affected then the interface will warn you and list the affected apps.

Reply:
Done.